### PR TITLE
Convert ingest processor supports ip type

### PR DESCRIPTION
**Pull request by gaobinlong**
### Description
In release 2.14.0, the convert ingest processor will support ip type, we need to document that.

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/6784

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by gaobinlong** [comment]
@hdhalter please help to review this PR, thank you!

**Review by vagimeli — APPROVED** [review]
Doc review complete. Ready for editorial.

File: _ingest-pipelines/processors/convert.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The following table lists the required and optional parameters for the `convert`
...
-`type`  | Required  | The type to convert the field value to. The supported types are `integer`, `long`, `float`, `double`, `string`, `boolean`, and `auto`. If the `type` is `boolean`, the value is set to `true` if the field value is a string `true` (ignoring case) and to `false` if  the field value is a string `false` (ignoring case). If the value is not one of the allowed values, an error will occur.  |
+`type`  | Required  | The type to convert the field value to. The supported types are `integer`, `long`, `float`, `double`, `string`, `boolean`, `ip`, and `auto`. If the `type` is `boolean`, the value is set to `true` if the field value is a string `true` (ignoring case) and to `false` if  the field value is a string `false` (ignoring case). If the type is set to `ip`, this processor validates whether the field value adheres to the correct format for IPv4 or IPv6 addresses; if the value is invalid, an error is raised. If the value is not one of the allowed values, an error will occur.  |
```
Comment (natebower) [inline _ingest-pipelines/processors/convert.md:38]:
```suggestion
`type`  | Required  | The type to convert the field value to. The supported types are `integer`, `long`, `float`, `double`, `string`, `boolean`, `ip`, and `auto`. If the `type` is `boolean`, then the value is set to `true` if the field value is a string `true` (ignoring case) and to `false` if  the field value is a string `false` (ignoring case). If the type is set to `ip`, then this processor validates whether the field value adheres to the correct format for IPv4 or IPv6 addresses; if the value is invalid, an error is raised. If the value is not one of the allowed values, an error will occur.  |
```

**Review by natebower — COMMENTED** [review]
@gaobinlong @vagimeli Just a couple of minor additions. Thanks!

**Pull request merged**